interoperability_report.py

```diff
@@ -359,461 +285,241 @@ def run_publisher_shape_main(
                         )
                     if index == 0:
                         produced_code[produced_code_index] = ReturnCode.OK
-                        # With these tests we check if we receive the data correctly,
-                        # in order to do it we are saving the samples sent
-                        if test_name == 'Test_Reliability_4' \
-                                    or test_name == 'Test_Ownership_4':
-                            for x in range(0, 80 ,1):
-                                pub_string = re.search('[0-9]{3} [0-9]{3}',
+                        max_samples_saved = 100 # used to save a max number of
```

Comment:
I dont know if it is worth it, because the code will be less readable and it is not so important to be able to change it

Reply:
I changed them in interoperability_report, but I dont know if I should do the same in test_suite, because the variables there are related to the functions, so it doesnt make a lot of sense to make them global no?